tests/chainer_tests/test_gradient_check.py

```diff
@@ -370,6 +370,55 @@ def f(x, y):
         gradient_check.check_backward(f, (x1, x2), g1, no_grads=[False, True])
 
 
+class TestCheckBackwardFailure(unittest.TestCase):
+
+    def _broken_func_1(self):
```

Comment:
Why did you wrap it with a function?

Reply:
Because they could be reused.
Indeed, `_broken_func_3` is used twice.

Reply:
ok